pkg/api/v20240812preview/platformworkloadidentityroleset_convert.go

```diff
@@ -57,9 +60,12 @@ func (c platformWorkloadIdentityRoleSetConverter) ToInternal(_new interface{}, o
 	out.Properties.OpenShiftVersion = new.Properties.OpenShiftVersion
 	out.Properties.PlatformWorkloadIdentityRoles = make([]api.PlatformWorkloadIdentityRole, 0, len(new.Properties.PlatformWorkloadIdentityRoles))
 
-	for i, r := range new.Properties.PlatformWorkloadIdentityRoles {
-		out.Properties.PlatformWorkloadIdentityRoles[i].OperatorName = r.OperatorName
-		out.Properties.PlatformWorkloadIdentityRoles[i].RoleDefinitionName = r.RoleDefinitionName
-		out.Properties.PlatformWorkloadIdentityRoles[i].RoleDefinitionID = r.RoleDefinitionID
+	for _, r := range new.Properties.PlatformWorkloadIdentityRoles {
+		role := api.PlatformWorkloadIdentityRole{
```

Comment:
Same comment as above.

Reply:
Similarly to the other one, I think this commit message address this concern: https://github.com/Azure/ARO-RP/pull/3608/commits/f8deb78edf13bd908393ed9e9a027cd177899747